src/docker/docker-compose.yml

```
@@ -1,10 +1,12 @@
 version: '3.7'
 services:
   # In Docker
+  # - TEST_POSTGRES_URI_MIGRATE=postgres://prisma:prisma@postgres/tests-migrate
   # - TEST_POSTGRES_URI=postgres://prisma:prisma@postgres/tests
   # - TEST_MYSQL_URI=mysql://prisma:prisma@mysql:3306/tests
   # - TEST_MARIADB_URI=mysql://prisma:prisma@mariadb:4306/tests
   # Outside Docker
+  # - TEST_POSTGRES_URI_MIGRATE=postgres://prisma:prisma@localhost:5432/tests-migrate
   # - TEST_POSTGRES_URI=postgres://prisma:prisma@localhost:5432/tests
   # - TEST_MYSQL_URI=mysql://prisma:prisma@localhost:3306/tests
```

This should be the new string in our local .envrc file now right? 
```suggestion
  # - TEST_MYSQL_URI=mysql://root:root@localhost:3306/tests
```
Just thought I would chime in as I'm setting up the new computer and my previously used environment variables were causing some tests to fail. Line 6 above would also seem to need the same change. 

I also get errors if I use localhost instead of 127.0.0.1, so I would actually propose changing the string to this:
```suggestion
  # - TEST_MYSQL_URI=mysql://root:root@127.0.0.1:3306/tests
```
What do you think? :) Do you also get errors if your connection string to mysql server uses localhost rather than 127.0.0.1?

---

Normally both `localhost` and `127.0.0.1` work.
If it doesn't it means that your host config was probably modified
Default is: (output from cat /etc/hosts)

```
##
# Host Database
#
# localhost is used to configure the loopback interface
# when the system is booting.  Do not change this entry.
##
127.0.0.1	localhost
255.255.255.255	broadcasthost
::1             localhost
```

But yes we could also default to 127.0.0.1 that sounds good 😄